### PR TITLE
cleaner main directory

### PR DESCRIPTION
**Pull request by dlaehnemann**
This new folder setup is meant to clean up the main folder and includes some more default folders. Hopefully, this makes setting up workflows and additional functionality (like schema validation and sample configuration) easier. This resulted from discussions with @johanneskoester and @christopher-schroeder.

### CODE REVIEW

**Comment by dlaehnemann** [comment]
A review would great, @johanneskoester.

File: {{ cookiecutter.repo_name }}/README.md [hunk 1]
```diff
@@ -50,30 +61,33 @@ If you not only want to fix the software stack but also the underlying OS, use
...
-2. [Clone](https://help.github.com/en/articles/cloning-a-repository) the fork to the desired working directory for the concrete project/run on your machine.
-3. [Create a new branch](https://git-scm.com/docs/gittutorial#_managing_branches) (the project-branch) within the clone and switch to it. The branch will contain any project-specific modifications (e.g. to configuration, but also to code).
-4. Modify the config, and any necessary sheets (and probably the workflow) as needed.
-5. Commit any changes and push the project-branch to your fork on github.
-6. Run the analysis.
-7. Optional: Merge back any valuable and generalizable changes to the [upstream repo](https://github.com/snakemake-workflows/{{cookiecutter.repo_name}}) via a [**pull request**](https://help.github.com/en/articles/creating-a-pull-request). This would be **greatly appreciated**.
-8. Optional: Push results (plots/tables) to the remote branch on your fork.
-9. Optional: Create a self-contained workflow archive for publication along with the paper (snakemake --archive).
-10. Optional: Delete the local clone/workdir to free space.
+### Step 7: Contribute back
```
Comment (johanneskoester) [inline {{ cookiecutter.repo_name }}/README.md]: May I ask you to also include the "update from upstream" step that is described in the dna-seq-varlociraptor workflow?

**Review by johanneskoester — CHANGES_REQUESTED** [review]
Awesome! One thing below. Once this is merged, we will probably migrate it into a pure github template repo.

File: {{ cookiecutter.repo_name }}/README.md (hunk 1 above)
Comment (dlaehnemann) [reply {{ cookiecutter.repo_name }}/README.md]: Done, also included an example `.github/workflow/main.yaml` for GitHub Actions.

**Comment by dlaehnemann** [comment]
@johanneskoester Please have another and feel free to merge.

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**